lisa/trace.py

```diff
@@ -3921,10 +3921,10 @@ def df_event(self, event, raw=None, window=None, signals=None, signals_init=True
 
         # Ensure the event name is set, as this sort of metadata might not be
         # saved in the swap
-        df.name = event
```

Comment:
And once more pandas gets in the way of reliable code. As the next line suggests, this was already left for backward compat only. The "new" way of adding custom metadata is by using `attrs` as done below:
https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.attrs.html

I'll make a fix that solves that problem in LISA, following which I expect we will be able to drop this patch in the PR.